src/quacc/recipes/newtonnet/ts.py

```diff
@@ -266,6 +282,269 @@ def quasi_irc_job(
     return relax_summary
 
 
+@job
+@requires(
+    has_newtonnet, "NewtonNet must be installed. Refer to the quacc documentation."
+)
+@requires(
+    has_geodesic_interpolate,
+    "geodesic-interpolate must be installed. Refer to the quacc documentation.",
+)
+def neb_job(
+    reactant_atoms: Atoms,
+    product_atoms: Atoms,
+    relax_job_kwargs: dict[str, Any] | None = None,
+    calc_kwargs: dict[str, Any] | None = None,
+    geodesic_interpolate_kwargs: dict[str, Any] | None = None,
+    neb_kwargs: dict[str, Any] | None = None,
+) -> NebSchema:
```

Comment:
I haven't commented on this previously but really should now:

In quacc, the `@job` is a discrete unit of work. A `@flow` is a workflow, concatenating multiple distinct `@job`s. Here (and in your other recipes, for what it's worth), you are actually doing a workflow. You have two relaxations (two "jobs") before running an NEB calculation. 

Normally, I would say this is not permitted. If we were running a Q-Chem calculation for instance, this would be highly undesirable because you would want the reactant and product relaxation jobs to be done concurrently, you'd want those results to be stored in your database, and if one fails you'd still like to store the results from the other. You can't do that if everything is packed in one `@job`.

The challenge here is that because NewtonNet and MLPs are so fast, it is tempting to pack everything into one `@job`. For users of workflow engines (like FireWorks) where each `@job` is a Slurm job, this means you'd be queuing more than you need. But not all workflow engines work that way (and even FireWorks has the option to not re-queue each job).

Jobflow does not support the `@flow` pattern, so I'm not going to inconvenience you by requiring that you implement it here. But I wanted to bring up this difference: what you are actually doing here is making a flow that combines steps from multiple jobs.

Reply:
I think the recipes like neb_ts_job and geodesic_ts_job will most likely go away. I have left them working right now but will probably remove them after a discussion together. I have put the workflows on a separate repository so things like the flow can be done over there. I have however not use jobflow over there so the parallel nature of the execution of the codes and also the fireworks related functionalities don't exist in those workflows as well.